api/src/main/java/org/openmrs/module/openconceptlab/Import.java

```diff
@@ -45,6 +45,10 @@
 	@Basic
 	@Column(name = "ocl_date_started")
 	private Date oclDateStarted;
+
+	@Basic
+	@Column(name = "last_downloaded_release")
+	private String lastDownloadedRelease;

```

Comment:
I asked to rename the variable to releaseVersion. As I understand it's not only downloaded, but release version imported for this import.